posts/2022-03-15-sql-retries-22003.adoc

```diff
@@ -143,6 +144,21 @@ Previously, a user would experience slower startup (around 4 or 5 seconds) if `m
 +
 A long running thread was added in 22.0.0.1 to gather CPU statistics. It was created as a non-daemon thread. This caused problems with stopping the server in products that embed Liberty and prevent the JVM from stopping until all non-daemon threads have exited. The server should stop cleanly. This problem was fixed by running the `CpuInfo` thread as a daemon.
 
+[#CVEs]
+== Security vulnerability (CVE) fixes in this release
+[cols="6*"]
+|===
+|CVE |CVSS Score |Vulnerability Assessment |Versions Affected |Version Fixed |Notes
+
+|http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-39038[CVE-2021-39038]
+|4.4
+|Clickjacking vulnerability
+|17.0.0.3 - 22.0.0.2
+|22.0.0.3
+|Affects the feature:openapi-3.1[], feature:mpOpenAPI-1.0[], feature:mpOpenAPI-1.1[] and feature:mpOpenAPI-2.0[] features
```

Comment:
looks like the macros don't work in the blogs?

Reply:
see features in table https://blogs-draft-openlibertyio.mybluemix.net/blog/2022/03/15/sql-retries-22003.html#CVEs